localstack/utils/bootstrap.py

```diff
@@ -295,12 +295,10 @@ def validate_localstack_config(name: str):
     image_name = ls_service_details.get("image", "")
     if image_name.split(":")[0] not in constants.OFFICIAL_IMAGES:
         warns.append(
-            'Using custom image "%s", we recommend using an official image: %s'
-            % (image_name, constants.OFFICIAL_IMAGES)
+            f'Using custom image "{image_name}", we recommend using an official image: {constants.OFFICIAL_IMAGES}'
         )
 
     # prepare config options
-    image_name = ls_service_details.get("image")
```

Comment:
nit: removed unused variable (duplicate from line 295 above, and not used further below)